packages/itwin/ec3-widget/src/components/LabelAction.scss

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import '~@itwin/itwinui-css/scss/variables';
+
+.label-details-container {
```

Comment:
Can you add ec3 in front of these selectors.

Reply:
ec3 added in front of the selectors